src/python/WMCore/WMBS/Subscription.py

```diff
@@ -182,14 +182,9 @@ def filesOfStatus(self, status, limit=0, loadChecksums=True, doingJobSplitting=F
 
         status = status.title()
         files = set()
-        if limit > 0:
-            action = self.daofactory(classname="Subscriptions.Get%sFilesByLimit" % status)
```

Comment:
@ticoann so, these DAOs could be under use, but we had not a single call to this method with the limit argument, that's why I just got rid of them :)